kernel/kernel-impl/src/main/webapp/WEB-INF/ignite-components.xml

```diff
@@ -27,6 +29,21 @@
         <property name="configuration" ref="org.sakaiproject.ignite.SakaiIgniteConfiguration"/>
     </bean>
 
+    <!--
+    Expiry policy to for when cache objects are considered expired (ttl)
+    Default is 60 minutes from when object is created
+    -->
+    <bean id="org.sakaiproject.ignite.cache.expiryPolicy"
+          class="javax.cache.expiry.CreatedExpiryPolicy"
+          factory-method="factoryOf">
+        <constructor-arg>
+            <bean class="javax.cache.expiry.Duration">
+                <constructor-arg value="MINUTES"/>
+                <constructor-arg value="60"/>
```

Comment:
I would propose a full day as caching time if right now is infinite.....

Reply:
I like the idea of scoping data to the session expiration also I was wanting to come up with more selective approach to configuring individual caches as well. How's 120 mins?

Reply:
Sure 120 mins works for me